howso/utilities/feature_attributes/pandas.py

```diff
@@ -185,7 +185,7 @@ def _get_random_value(self, feature_name: str, no_nulls: bool = False) -> Option
         cases = self.data[feature_name]
         if no_nulls:
             cases = cases.loc[~self.data[feature_name].isnull()]
-        if len(cases) <= 1:
+        if len(cases) < 1:
```

Comment:
This will now raise a ValueError due to the filter on the next line. If length is 1 we should just return it directly, if length is 0 return None, otherwise do the random filter.